### ISSUE TITLE
Ea Sports Photo Game Face Marker Placement Error bug?

### ISSUE THREAD

**Issue by setho123113** [RPCS3/rpcs3#12211]
Hello, I am Seth.
I am having an issue regarding Ea Sports Face-Scanning or also known as "Ea Sports Photo GameFace"
the error says,

"The face generation cannot finish. please review your marker placement and photos and try again." 

in this photo I just linked. it shows the error that I got.

![FaceBreakerBug](https://user-images.githubusercontent.com/81538265/173191739-3221211c-3e14-4804-a70d-ec97e4ae2fb9.png)

I also had a similar issue with the Fight Night Games.

here's the log from RPCS3.

https://drive.google.com/file/d/1s5d9tls0pqWm-WTpzDYOJXuonDCcAfyl/view?usp=sharing

I hope you guys can hopefully fix this issue. because I would love to scan me and my friend's faces for the games.

I'm really looking forward to it.

from, 
Seth

**Comment by Megamouse** [comment]
Before we investigate this, please try with different lighting and compare to a real PS3.

**Comment by setho123113** [comment]
> Before we investigate this, please try with different lighting and compare to a real PS3.

Well.. I think FaceBreaker's Photo Scanning feature does that lighting thing you mentioned when the error pops up. (I don't know since I am new to this whole PS3 Emulation thing) but the PS3 is built differently with real hardware. such as NVIDIA, and the central processing unit is based off of Rambus.

**Comment by Megamouse** [comment]
what i mean is, you should try with a brighter lamp in your room.
and you should try on the PS3 if the image looks the same.

**Comment by setho123113** [comment]
> what i mean is, you should try with a brighter lamp in your room. and you should try on the PS3 if the image looks the same.

Yeah. I should try. because I have a lamp. but unfortunately. about the PS3. I don't have one anymore due to it breaking years ago. but the games I have are games I previously had on the Xbox 360. which I wanted to compare the differences. some PS3 games have more content on the PS3 versions than on the 360 which is rather interesting.

**Comment by digant73** [comment]
@setho123113 is that still an issue?

**Comment by digant73** [comment]
@setho123113 please provide an update on this issue

**Comment by setho123113** [comment]
> [@setho123113](https://github.com/setho123113) please provide an update on this issue

Luckily I managed to fix it since then. The thread can be closed now.

**Comment by digant73** [comment]
@setho123113 please close the ticket. Also, It could help if you provide some info about how you fixed the issue
